test/lhttpc_tests.erl

```diff
@@ -407,6 +409,30 @@ connection_timeout() ->
     timer:sleep(100),
     ?assertEqual({0,0}, lhttpc_lb:connection_count("localhost", Port, false)).
 
+-define(REQUEST_LIMIT, 3).
```

Comment:
Seems a bit strange to make this value global, since it only really has meaning to the test.  Could be:
```
request_limit() ->
    RequestLimit = 3,
    Port = start(gen_tcp, lists:duplicate(?REQUEST_LIMIT, fun simple_response/5)),
...
```
?